packages/components/pie-modal/src/modal.scss

```diff
@@ -91,6 +91,11 @@
 
                 // In this case, the modal must exceed the previous maximum width
                 --modal-max-inline-size: 100%;
+
+                // this ensures the content container spans the full height
+                > .c-modal-scrollContainer {
+                    block-size: 100%;
```

Comment:
Do you need to include the writing mode with these values at all?

Reply:
i don't think it's necessary as we don't support different writing modes yet 💯